erpnext_sync.py

```diff
@@ -182,15 +182,16 @@ def send_to_erpnext(employee_field_value, timestamp, device_id=None, log_type=No
     url = f"{config.ERPNEXT_URL}/api/method/{endpoint_app}.hr.doctype.employee_checkin.employee_checkin.add_log_based_on_employee_field"
     headers = {
         'Authorization': "token "+ config.ERPNEXT_API_KEY + ":" + config.ERPNEXT_API_SECRET,
-        'Accept': 'application/json'
+        'Accept': 'application/json',
+        'Content-Type': 'application/json'
```

Comment:
```suggestion
        'Accept': 'application/json'
```

If you use `json`, setting content type header is not really required.


![image](https://github.com/frappe/biometric-attendance-sync-tool/assets/9079960/7b535091-8a3d-46f6-a1a8-f08597398a6a)